src/main/charts/confluence/templates/_helpers.tpl

```diff
@@ -149,6 +149,10 @@ app.kubernetes.io/instance: {{ .Release.Name }}
 -Datlassian.logging.cloud.enabled={{.Values.fluentd.enabled}}
 {{- end }}
 
+{{- define "confluence.sysprop.enable.synchrony.by.default" -}}
+-Dsynchrony.by.default.enable.collab.editing.if.manually.managed=true
```

Comment:
What would happen if this is used in Confluence that doesn't support the system property? Nothing?

Reply:
Nothing will happen, as old Connie doesn't check that property